.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
```

Comment:
Ideally, stuff like `.idea` or other OS/IDE specific files should be located in a machine global gitignore file, rather than bound to a repository-specifc `.gitignore`. See [this tutorial](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) on how to set it up. In any case, we can remove it later on from this repo, it's not crucial :)